### PR TITLE
Hand grab calcs

### PR DESCRIPTION
**Pull request by ericrius1**
…jects

This PR adds some logic to the handGrab script that takes the avatar hand size into consideration when calculating the positional offset of entities. This is not a perfect solution, but it does somewhat improve the positioning of objects in avatar hands that are different sizes than a more standard hand (which was taken to be Owen)

### CODE REVIEW

**Comment by ericrius1** [comment]
Test script: https://rawgit.com/ericrius1/hifi/handGrabCalcs/examples/controllers/handControllerGrab.js

**Comment by hifi-gustavo** [comment]
Build console output for commit: cd983db4196260bbe85a9e058559cc13ee33fa3e available at: https://gist.github.com/e4cd60451001961cbd77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

File: unpublishedScripts/hiddenEntityReset.js [hunk 1]
```diff
@@ -169,7 +169,7 @@
         }
 
         function createRaveStick(position) {
-            var modelURL = "https://s3.amazonaws.com/hifi-public/eric/models/rave/raveStick.fbx";
+            var modelURL = "https://s3-us-west-1.amazonaws.com/hifi-content/eric/models/raveStick.fbx";
```
Comment (imgntn) [inline unpublishedScripts/hiddenEntityReset.js]: haven't seen the region specified like this before -- i usually have a link like  http://hifi-content.s3.amazonaws.com/james/avatars/Jack_Skellington.fst

**Comment by imgntn** [comment]
@ericrius1 should your changed to the rave stick relative position be in its individual file as well as the toybox resetters?

File: unpublishedScripts/hiddenEntityReset.js (hunk 1 above)
Comment (ericrius1) [inline unpublishedScripts/hiddenEntityReset.js]: This is the link I got when uploading the the hifi-content bucket

**Comment by ericrius1** [comment]
@imgntn - yes, good call

**Comment by hifi-gustavo** [comment]
Build console output for commit: 24796c1a1eb46cce988752634bd40ba62cf28bdd available at: https://gist.github.com/e4cd60451001961cbd77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4b50a030e06aeffd105abd977511e4f1932edf30 available at: https://gist.github.com/e4cd60451001961cbd77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4f80753d7b0c9ec88f49cc3ea3034d5929967e9f available at: https://gist.github.com/3afd7c7d11c512294a23<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c4d7b7b9bb2afc5abd5f084c45391ef51a372fca available at: https://gist.github.com/e4cd60451001961cbd77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 679070ff6e1b155fcb8388aa3c446515499e4d8f available at: https://gist.github.com/285d77fab3676da7d9c3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6736.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6736.dmg

**Pull request merged**